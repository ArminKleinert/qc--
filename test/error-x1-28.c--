const     true = 1 == 1;

foo() {
    jump true ();
}
