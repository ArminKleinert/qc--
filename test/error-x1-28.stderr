error-x1-28.c--:8.1 Error: jump target is not a pointer
